rocAL/rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -22,16 +22,11 @@
 import rocal_pybind as b
 from amd.rocal.pipeline import Pipeline
 
-def image(*inputs, user_feature_key_map = None, path='', file_root ='', annotations_file= '', shard_id = 0, num_shards = 1, random_shuffle = False, affine=True, bytes_per_sample_hint=0, cache_batch_copy= True, cache_debug = False, cache_size = 0, cache_threshold = 0,
-                 cache_type='', device_memory_padding=16777216, host_memory_padding=8388608, hybrid_huffman_threshold= 1000000, output_type = types.RGB,
-                 preserve=False, seed=-1, split_stages=False, use_chunk_allocator= False, use_fast_idct = False, device = None):
+def image(*inputs, user_feature_key_map=None, path='', file_root='', annotations_file='', shard_id=0, num_shards=1, random_shuffle=False, affine=True, bytes_per_sample_hint=0, cache_batch_copy=True, cache_debug=False, cache_size=0, cache_threshold=0,
+          cache_type='', device_memory_padding=16777216, host_memory_padding=8388608, hybrid_huffman_threshold=1000000, output_type=types.RGB, decoder_type=types.DECODER_TJPEG,
+          preserve=False, seed=1, split_stages=False, use_chunk_allocator=False, use_fast_idct=False, device=None):
     reader = Pipeline._current_pipeline._reader
-    if (device == "gpu"):
```

Comment:
Why are we removing this? Is the device argument set for decoder?

Reply:
added it back